tests/__snapshots__/CarrierTest__it_instantiates_carriers_from_name_with_(belgie)__1.json

```diff
@@ -1,86 +1,123 @@
 [
-  {
-    "id": 1,
-    "name": "postnl",
-    "human": "PostNL",
-    "enabled": true,
-    "isDefault": true,
-    "optional": false,
-    "primary": false,
-    "type": "main",
-    "capabilities": {
-      "deliveryTypes": ["standard", "pickup"],
-      "features": {
-        "labelDescriptionLength": 45,
-        "multiCollo": true
-      },
-      "packageTypes": ["package"],
-      "shipmentOptions": {
-        "signature": true,
-        "ageCheck": false,
-        "return": false,
-        "onlyRecipient": true,
-        "largeFormat": true,
-        "insurance": [0, 50000]
-      }
+    {
+        "id": 1,
+        "name": "postnl",
```

Comment:
je git hooks zijn niet gerund, hier moet nog prettier overheen

als het goed is moet je alleen maar 1x `husky install` in je repo doen en zou het vervolgens wel moeten werken op iedere commit 🤔

voor nu kan je even dit doen om het te fixen

```sh
prettier --write tests/__snapshots__/*.json
```